lib/web_ui/lib/src/engine/html/color_filter.dart

```diff
@@ -10,15 +10,17 @@ import '../dom.dart';
 import '../embedder.dart';
 import '../svg.dart';
 import '../util.dart';
+import '../vector_math.dart';
 import 'bitmap_canvas.dart';
 import 'path_to_svg_clip.dart';
 import 'shaders/shader.dart';
 import 'surface.dart';
+import 'surface_stats.dart';
 
 /// A surface that applies an [ColorFilter] to its children.
 class PersistedColorFilter extends PersistedContainerSurface
     implements ui.ColorFilterEngineLayer {
-  PersistedColorFilter(PersistedColorFilter? super.oldLayer, this.filter);
+  PersistedColorFilter(PersistedColorFilter? super.oldLayer, this.filter, this.offset);
```

Comment:
@yjbanov do you have any pointers on this? I tried to base it off of the ImageFilterLayer change but this class works a bit differently